.github/workflows/code_checks.yml

```diff
@@ -75,6 +75,17 @@ jobs:
                     coverage: none # disable xdebug, pcov
             -   run: composer install --no-progress
             -   run: composer rector
+    rector-ci:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -
+                uses: shivammathur/setup-php@v1
+                with:
+                    php-version: 7.3
+                    coverage: none # disable xdebug, pcov
+            -   run: composer install --no-progress
+            -   run: composer rector-ci
```

Comment:
You remove the `rector` job above. It's duplicated now and only with `--dry-run` it actually fails the CI

Reply:
I'll handle it